.ci/lib.sh

```diff
@@ -417,3 +417,24 @@ get_pr_changed_file_details()
 {
 	get_pr_changed_file_details_full | grep -v "vendor/"
 }
+
+# Receives as list of files and/or, build a sha256sum from their contents
+sha256sum_from_files() {
+	# TODO: implement me!
+	echo "zzzzzzzzzzzzzzzzz"
+
+}
+
+gen_qemu_files_sha256sum() {
+	local pkg_dir="${kata_repo_dir}/tools/packaging"
+	# Files and dirs
+	files="${pkg_dir}/qemu \
+		${pkg_dir}/static-build/qemu \
+		${pkg_dir}/static-build/qemu/qemu.blacklist \
+		${pkg_dir}/static-build/scripts \
+		${pkg_dir}/scripts"
```

Comment:
I'd also add the QEMU version, coming from the version file, to the dance.

Right now we have patches which would accurately get any new change, but if we ever happen to use vanilla QEMU, not having the version could mislead us.

Reply:
Good catch @fidencio ! I will add that file too.